tests/test_cvc.ml

```diff
@@ -0,0 +1,109 @@
+open OUnit2
+
+let rsa_suite =
+  let open Key_parsers in
+  let open Asn1.RSA_CVC in
+  let expected_public =
+    let n =
+      Z.of_string
+        "0xFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFF\
```

Comment:
Using this modulus you won't be able to detect string reversal (e.g. the test suite will pass if you remove the string reversal part). You can use, in order of preference:

-`00 01 02 03...7f 80`
-a random sequence of bytes
-an actual modulus generated for example by openssl. I can provide one if you'd like.